cmd/commands/runtime.go

```diff
@@ -723,30 +728,38 @@ func installComponents(ctx context.Context, opts *RuntimeInstallOptions, rt *run
 	if err = createReporter(
 		ctx, opts.InsCloneOpts, opts, reporterCreateOptions{
 			reporterName: store.Get().WorkflowReporterName,
-			resourceName: store.Get().WorkflowResourceName,
-			group:        argowf.Group,
-			version:      argowf.Version,
-			saName:       store.Get().CodefreshSA,
+			gvk: []Gvk{
+				{
+					resourceName: store.Get().WorkflowResourceName,
+					group:        "argoproj.io",
+					version:      "v1alpha1",
+				},
+			},
+			saName: store.Get().CodefreshSA,
 		}); err != nil {
 		return fmt.Errorf("failed to create workflows-reporter: %w", err)
 	}
 
-	if err = createReporter(ctx, opts.InsCloneOpts, opts, reporterCreateOptions{
-		reporterName: store.Get().ReplicaSetReporterName,
-		resourceName: store.Get().ReplicaSetResourceName,
-		group:        "apps",
-		version:      "v1",
-		saName:       store.Get().ReplicaSetReporterServiceAccount,
-	}); err != nil {
-		return fmt.Errorf("failed to create replicaset-reporter: %w", err)
-	}
-
 	if err = createReporter(ctx, opts.InsCloneOpts, opts, reporterCreateOptions{
 		reporterName: store.Get().RolloutReporterName,
-		resourceName: store.Get().RolloutResourceName,
-		group:        "argoproj.io",
-		version:      "v1alpha1",
-		saName:       store.Get().RolloutReporterServiceAccount,
+		gvk: []Gvk{
+			{
```

Comment:
do you think it's needed in this case to replace these hard coded strings with values defined in the store?

Reply:
ideally, we will get these values from the matching argo-rollouts/argo-workflows/argo-whatever types.go
i don't remember if i looked for them, and couldn't find them, or we just decided at the end to not add a ton of import statements and dependencies, just to get a few strings.
i don't think that every string needs to be in the `store.go` file, personally.